### ISSUE TITLE
Unable to modify ports for predefined service

### ISSUE THREAD

**Issue by piotrszlenk** [zscaler/terraform-provider-zia#262]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

Terraform version 1.4.4

```
terraform -v
Terraform v1.4.4
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v4.56.0
+ provider registry.terraform.io/hashicorp/vault v3.19.0
+ provider registry.terraform.io/zscaler/zia v2.4.6

Your version of Terraform is out of date! The latest version
is 1.5.5. You can update by downloading from https://www.terraform.io/downloads.html
```

### Affected Resource(s)

Resource: zia_firewall_filtering_network_service

### Terraform Configuration Files

```hcl
resource "zia_firewall_filtering_network_service" "this" {
    description        = "DHCP_DESC"
    name               = "DHCP"
    tag                = "DHCP"
    is_name_l10n_tag = true
    type               = "PREDEFINED"

    dest_tcp_ports {
        end   = 68
        start = 67
    }

    dest_udp_ports {
        end   = 68
        start = 67
    }

    dest_tcp_ports {
        end   = 70
        start = 69
    }

    dest_udp_ports {
        end   = 70
        start = 69
    }
}
```

### Debug Output

https://gist.github.com/piotrszlenk/027f30b8eaf24319e5763d784fd4fdb4

### Expected Behavior

Add custom ports to predefined service.

### Actual Behavior

I am unable to modify ports of predefined network service. 

```
╷
│ Error: FAILED: POST, https://zsapi.zscalerbeta.net/api/v1/networkServices, 400, 400 , {"code":"DUPLICATE_ITEM","message":"DUPLICATE_ITEM"}, api responded with code: 400
│
│   with zia_firewall_filtering_network_service.this,
│   on main.tf line 46, in resource "zia_firewall_filtering_network_service" "this":
│   46: resource "zia_firewall_filtering_network_service" "this" {
│
│ 
```

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

I am experimenting with zscalerbeta.net. I can't tell if this works on regular tenants as I have no access to them.

**Comment by willguibr** [comment]
Hi @piotrszlenk 
The issue is that you are trying to create a predefined service instead of modifying; hence the reason you're receiving DUPLICATED_ITEM because there's a service with the same name already. So, please follow this procedure:
1. Create your HCL configuration with an empty block for the resource 
2. Import the service by using` terraform import zia_firewall_filtering_network_service.this "DHCP"`
3. Run Terraform show
4. Copy the output and place in your HCL
5. Remove computed values i.e id.
6. Run terraform apply (You shouldn't modify anything at that point)
7. Apply the modifications you want. Ports, names etc. and run terraform apply again.

In other words, this issue is all because there's a service with the same name already defined in your tenant

Regards,
William
